serverlessworkflow/modules/ROOT/pages/getting-started/java-embedded-workflows.adoc

```diff
@@ -78,11 +84,37 @@ Therefore, you can modify the previous example to generate the same output when
 <6> Build the workflow definition
 <7> Execute and print as in previous example
 
-
-== Further reading
+=== Additional fluent examples
 
 You can find additional and commented examples of fluent API usage (including jq expression evaluation and orchestration of rest services) link:{kogito_sw_examples_url}/sonataflow-fluent[here]
 
+== Dependencies explanation
+
+In order to keep the number of required dependencies as smaller as possible, embedded workflow uses a modular approach. The rationale is to avoid adding to the dependency set something that you are not going to use. For example, OpenAPI module is based on Swagger parser, if you are not going to call any OpenAPI service, it is better to avoid adding Swagger to the dependency set. This means the link:{swf_executor_core_maven_repo_url}[Core] dependency does not include the stuff to use gRPC, OpenAPI or most custom function types.  
```

Comment:
rephrase and typo

```suggestion
The embedded workflow uses a modular approach to keep the number of required dependencies as small as possible. The rationale is to avoid adding something that you are not going to use to the dependency set. This means the link:{swf_executor_core_maven_repo_url}[Core] dependency does not include the stuff to use link:{spec_doc_url}#using-functions-for-rpc-service-invocations[gRPC] (which will add the Guava libraries to the dependency set) or https://github.com/serverlessworkflow/specification/blob/0.8.x/specification.md#using-functions-for-restful-service-invocations[OpenAPI] (which will add Swagger libraries to the dependency set) function types. They need to be included explicitly.
```typos

```suggestion
The embedded workflow uses a modular approach to keep the number of required dependencies as small as possible. The rationale is to avoid adding something that you are not going to use to the dependency set. This means the link:{swf_executor_core_maven_repo_url}[Core] dependency does not include the stuff to use link:{spec_doc_url}#using-functions-for-rpc-service-invocations[gRPC] (which will add the Guava libraries to the dependency set) or https://github.com/serverlessworkflow/specification/blob/0.8.x/specification.md#using-functions-for-restful-service-invocations[OpenAPI] (which will add Swagger libraries to the dependency set) function types. They need to be included explicitly.
```

Reply:
Sorry @gmunozfe , I changed the phrasing while you were reading it, can you review it again?